### ISSUE TITLE
can't Clean cache on 1.0.1.10

### ISSUE THREAD

**Issue by Ghost-Girls** [QuadSpinner/Highlighter#13]
where is .vs\Highlights.xml ? search by everything, whithout any result.

**Comment by daxpandhi** [comment]
This file is made only when `SolutionTags` options are used. Otherwise the options are stored in Visual Studios' options in the registry or local VS settings file based on your version.

Do you want to reset the settings?

**Comment by Ghost-Girls** [comment]
> This file is made only when `SolutionTags` options are used. Otherwise the options are stored in Visual Studios' options in the registry or local VS settings file based on your version.只有当选项被使用时，这个文件才会被创建。否则，这些选项将存储在注册表中的Visual Studio选项中，或者基于你的版本，存储在本地VS设置文件中。
> 
> Do you want to reset the settings?你想要重置设置吗？

Yes, I created a branch and found that the generated extension conflicted with the previous extension after installation.
Currently, the conflict was resolved after I deleted the tag list.
